apps/bbsync/save.py

```diff
@@ -0,0 +1,59 @@
+from collectors.bzimport.collectors import BugzillaConnector
+from osidb.models import Flaw
+
+from .query import BugzillaQueryBuilder
+
+
+class BugzillaSaver(BugzillaConnector):
+    """
+    Bugzilla flaw bug save handler
+    flaw validity is assumed and not checked
+    """
+
+    class UnsaveableFlaw(Exception):
+        """
+        error caused by attempt to save a flaw which cannot be saved
+        either by its nature or due to the current saver capabilities
+        """
+
+        pass
+
+    def __init__(self, flaw):
+        """
+        init stuff
+        """
+        self.flaw = flaw
+
+    def save(self):
+        """
+        generic save serving as class entry point
+        which calls create or update handler to continue
+        returns an updated flaw instance (without saving)
+        """
+        return self.create() if self.flaw.bz_id is None else self.update()
+
+    def create(self):
+        """
+        create flaw in Bugilla
+        """
+        bqb = BugzillaQueryBuilder(self.flaw)
+        response = self.bz_conn.createbug(bqb.query)
+        self.flaw.meta_attr["bz_id"] = response.id
```

Comment:
I would hope that this doesn't change between the syncs, seems odd to do it

Reply:
This is the creation so there is no Bugzilla ID before. I am not yet sure how will the save logic work and of course it can be done by the sync but for now it makes sense to me to update the flaw model with the new Bugzilla ID.